### ISSUE TITLE
How many connections for hivemq free edition?

### ISSUE THREAD

**Issue by hanyou** [sitewhere/sitewhere#80]
free hivemq edition support connections?

**Comment by derekadams** [comment]
The free edition supports 20 connections. You can also use another broker such as Mosquitto if you want to scale beyond that number of connections without paying for it.

**Comment by hanyou** [comment]
how to deploy it, we need some deployment document.

**Comment by derekadams** [comment]
HiveMQ has complete documentation at the following URL:

http://www.hivemq.com/docs/hivemq/latest/

Is that what you were looking for?

**Comment by hanyou** [comment]
no, i need all sitewhere deployment ducument for version 1.8, i need know how to work with Mosquitto together?

**Comment by erliuwang** [comment]
I also need to know about this issues,help please @

**Comment by derekadams** [comment]
You should be able to use any standard MQTT broker (including Mosquitto) with SiteWhere. Is there anything in particular that is not working?
